pkg/cri/server/image_verifier/v1/image_verifier.proto

```diff
@@ -0,0 +1,34 @@
+/*
+	Copyright The containerd Authors.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+syntax = "proto3";
+
+package containerd.cri.image_verifier;
+option go_package = "github.com/containerd/containerd/pkg/cri/server/image_verifier/v1;image_verifier";
+
+service image_verifier {
+  rpc VerifyImage(VerifyImageRequest) returns (VerifyImageResponse);
+}
+
+message VerifyImageRequest {
+  string image_name = 1;
+  string image_digest = 2;
+}
+
+message VerifyImageResponse {
+  bool ok = 1;
+  string reason = 2;
+}
```

Comment:
These fields can be just included in the error message, so probably `VerifyImageResponse` can be empty

Reply:
My reasoning here is that it may be desirable to have a "fail open" setting in the CRI, where errors in communicating with the image verifier plugin result in allowed images. If images are rejected by returning an error, then there is no way for the CRI to distinguish a rejected image from an error.